src/state/ledger/command.rs

```diff
@@ -31,59 +33,106 @@ pub enum Command {
     Delegation(Delegation),
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
```

Comment:
we can derive `Hash` here too, might as well

Reply:
Let's do it when we need it. Nothing warrants it yet AFAICT